pkgs/data/icons/nordzy-icon-theme/default.nix

```
@@ -0,0 +1,56 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+, gtk3
+, nordzy-theme-name ? "Nordzy"
+, nordzy-themes ? [ "all" ] # Override this to only install selected themes
+}:
+
+let
+  themes-arg-string = lib.strings.concatMapStrings (theme: "-t ${theme} ") nordzy-themes;
+in
+stdenvNoCC.mkDerivation {
+  pname = "nordzy-icon-theme";
+  version = "unstable-2021-12-14";
```

```suggestion
  version = "unstable-2022-01-15";
```
Use the latest revision.